modules/xhbBidAdapter.md

```diff
@@ -0,0 +1,103 @@
+# Overview
+
+```
+Module Name:  XHB Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer:   daniel.hoffmann@xaxis.com 
+```
+
+# Description
+
+Connects to Appnexus exchange for bids.
+
+XHB bid adapter supports Banner, Video (instream and outstream) and Native.
+
+# Test Parameters
+```
+var adUnits = [
+   // Banner adUnit
+   {
+       code: 'banner-div',
+       sizes: [[300, 250], [300,600]],
+       bids: [{
+         bidder: 'xhb',
+         params: {
+           placementId: '10433394'
+         }
+       }]
+   },
+   // Native adUnit
+   {
+      code: 'native-div',
+      sizes: [[300, 250], [300,600]],
+      mediaTypes: {
+        native: {
+          title: {
+            required: true,
+            len: 80
+          },
+          body: {
+            required: true
+          },
+          brand: {
+            required: true
```

Comment:
In the test placement - the `brand` is not included which is preventing the bid from being accepted by the native param checks.  If you want the `brand` to be a required field in general for your publishers, could you update the test ad unit to include this field?  If the field isn't strictly required, could you update this .md file?

Reply:
Actually, I just copied this from the AN adapter, I assume that one has the issue?

Reply:
Yeah, we don't need that to be required.
Can I just set it to false in the md file?

Reply:
That's fine, or you could just remove the `brand` object entirely.